lib/data/school_mass_importer.rb

```diff
@@ -1,9 +1,10 @@
-require 'breasal'
-require 'activerecord-import'
+require_relative 'school_data_helpers'
```

Comment:
I think this should be under `app/models/bookings` - it is a specific item for manipulating our data?